code/src/layouts/app-header/components/categories/Categories.scss

```diff
@@ -0,0 +1,17 @@
+.menu {
+  display: flex;
+  justify-content: flex-start;

```

Comment:
Is it required here?

Reply:
> Is it required here?

Yeap.
Because menu items should be `flex-start` under header according to the design